pywinauto/win32structures.py

```diff
@@ -41,9 +41,9 @@
 from . import sysinfo
 
 
-class Structure(Struct):
+class StructureMixIn:

```

Comment:
Every class should be inherited at least from `object` to be "new-style class" in Python 2.7.